### DISCUSSION TITLE
Feature Request: Tab Sets

### DISCUSSION THREAD

**Discussion by helicos6** [rxhanson/RectanglePro-Community#245] [Ideas]
There is no mac app that can do what TidyTabs for Windows does. At least nothing that works properly and maintained.

https://www.nurgo-software.com/products/tidytabs

Being able to group windows of an app that doesn't support tabs or group windows of different apps into a tab set can greatly increase window management and productivity. It's definitely something that would make Rectangle stand out.

**Comment by rxhanson** [comment]
Thanks for the feedback. I have a pretty large backlog already, but I'll give this some thought. It might somehow be able to tie into some of the future features I have planned for the Stash functionality in Rectangle Pro.

**Comment by helicos6** [comment]
@rxhanson That would be great, thanks!
